compiler/src/dotty/tools/dotc/tastyreflect/FromSymbol.scala

```diff
@@ -1,37 +1,40 @@
 package dotty.tools.dotc.tastyreflect
 
 import dotty.tools.dotc.ast.tpd
-import dotty.tools.dotc.core.Contexts.Context
 import dotty.tools.dotc.core.Flags._
+import dotty.tools.dotc.core.StdNames._
 import dotty.tools.dotc.core.Symbols._
+import dotty.tools.dotc.core.Types._
 
-object FromSymbol {
+trait FromSymbol { tasty: TastyImpl =>
```

Comment:
It seems `tasty: TastyImpl` is not used in the implementation? What's the motivation to change from `object` to `trait`?

Reply:
This change is not needed, I will revert it

Reply:
Reverted